src/main/java/com/microsoft/sqlserver/jdbc/IOBuffer.java

```diff
@@ -2925,9 +2989,30 @@ private Socket getDefaultSocket(String hostName, int portNumber, int timeoutInMi
             InetSocketAddress cacheEntry = SQLServerConnection.getDNSEntry(hostName);
             addr = (null != cacheEntry) ? cacheEntry : addr;
         }
+
         return getConnectedSocket(addr, timeoutInMilliSeconds);
     }
 
+    private Queue<InetAddress> getIPv6Addresses(InetAddress[] addresses) {
```

Comment:
the names seem a bit misleading, this is actually returning an ipv6AdressQueue

Reply:
Agreed, changes made.